src/nile/common.py

```diff
@@ -39,11 +43,7 @@ def get_gateways():
 
     except FileNotFoundError:
         with open(NODE_FILENAME, "w") as f:
-            networks = {
-                "localhost": "http://127.0.0.1:5050/",
-                "goerli2": "https://alpha4-2.starknet.io",
-                "integration": "https://external.integration.starknet.io",
-            }
+            networks = {"localhost": "http://127.0.0.1:5050/", **OTHER_NETWORKS}
```

Comment:
If we add DEFAULT_GATEWAYS to queries even when the node.json exists:

```suggestion
            networks = {"localhost": "http://127.0.0.1:5050/"}
```